packages/babel-plugin-redux-saga/test/__snapshots__/runner.test.js.snap

```diff
@@ -0,0 +1,976 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`babel v6 attach source to declaration 1`] = `
```

Comment:
hm, those snapshots are imho less readable. Would it be possible to go back to separate files and suffix expected files with babel's version?

Reply:
if it's less readable for other maintainers, I'll change the structure 👍

Reply:
I'd appreciate the change, imho separate files are easier to browse & inspect + syntax highlighting works in editors

Reply:
done